src/ProcessManager/ChromeManager.php

```diff
@@ -32,7 +32,7 @@
 
     public function __construct(?string $chromeDriverBinary = null, ?array $arguments = null, array $options = [])
     {
-        $this->process = new Process([$chromeDriverBinary ?? $this->findChromeDriverBinary()], null, null, null, null);
+        $this->process = new Process([$chromeDriverBinary ?: $this->findChromeDriverBinary()], null, null, null, null);
```

Comment:
Why this change?

Reply:
To avoid `$chromeDriverBinary` be an empty string 😄

Reply:
How can it be?

Reply:
With a simple developer mistake, I know I might be paranoid, but if you think I should revert this, feel free to ask, and I will

Reply:
No it's ok! Can you just document this new env var in the README file please?